proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -280,6 +280,11 @@ message WorkflowExecutionInfo {
     // Should be deprecated once the transition history is fully launched
     repeated VersionedTransition previous_transition_history = 100;
     VersionedTransition last_transition_history_break_point = 101;
+
+    // This is a set of child workflows that were initialized after the reset point in the parent workflow.
+    // The children are identified by the key "workflow_type:workflow_id". When the parent starts to make progress after reset, it uses this data to
```

Comment:
we don't have any restrictions on the char used in the workflowID & workflowType I think, so they could conflict. But i guess it doesn't matter too much in reality.

Reply:
Yes. It should not be an issue since we are not parsing the value back. it's kind of like a hash. So even if they include special chars or `:` char, it should be ok.

Reply:
oh I mean the hash could conflict for different workflowType and workflowID.

Reply:
ah! yes. There is a possibility of that. Maybe in future iterations we can build a good and signature.